pypi_push.sh

```diff
@@ -0,0 +1,9 @@
+##STOP - Make sure you updated the version in the setup.py to match the version in github to be uploaded.
```

Comment:
is it typically for the versioning to be a manual process, or are there automated tools? I am not suggesting that we need to do this now, just curious?